bananas_api/new_upload/readers/heightmap.py

```diff
@@ -47,14 +51,11 @@ def read(self, fp):
 
         try:
             im = Image.open(fp)
+        except Image.DecompressionBombError:
+            raise ValidationException("Image is too large.")
```

Comment:
`If the number of pixels is greater than twice [MAX_IMAGE_PIXELS](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.MAX_IMAGE_PIXELS), then a DecompressionBombError will be raised instead.`

I was wondering why my 20kx20k map wasn't triggering this, but I have to go slightly bigger before it does so :) Seems we need both statements or something to make this work properly :)